pom.xml

```diff
@@ -297,21 +297,16 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
     <aspectj.version>1.9.7</aspectj.version>
     <aspectj-plugin.version>1.14.0</aspectj-plugin.version>
     <restrict-imports.enforcer.version>2.4.0</restrict-imports.enforcer.version>
-    <bzip2.version>1.0.8</bzip2.version>
-    <zlib.version>1.2.13</zlib.version>
-    <lz4.version>1.9.3</lz4.version>
-    <snappy.version>1.1.8</snappy.version>
-    <zstd.version>1.4.9</zstd.version>
     <jgrapht.version>1.0.1</jgrapht.version>
 
     <spring.version>5.3.27</spring.version>
     <jooq.version>3.11.10</jooq.version>
 
     <vault.driver.version>5.1.0</vault.driver.version>
     <native.lib.tmp.dir></native.lib.tmp.dir>
+    <properties.maven.plugin.version>1.0.0</properties.maven.plugin.version>
```

Comment:
Why are we using an old version with 3 known CVEs?
https://mvnrepository.com/artifact/org.codehaus.mojo/properties-maven-plugin/1.0.0

The latest is 1.2.1 but it still introduces transitive dependency with a known CVE:
https://mvnrepository.com/artifact/org.codehaus.mojo/properties-maven-plugin/1.2.1

Reply:
Yeah try that. See if it breaks anything